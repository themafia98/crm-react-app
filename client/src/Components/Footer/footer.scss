footer {
    display: flex;
    justify-content: center;
    align-items: center;
    background-color: rgb(23, 23, 23);
    height: 15vh;
    .footer__title {
        font-weight: bold;
        color: #ffffff;
    }
}