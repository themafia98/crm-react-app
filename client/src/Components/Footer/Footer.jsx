import React from 'react';
import PropTypes from 'prop-types';
import './footer.scss';
const Footer = ({footerTitle}) => {
    return (
        <footer>
            <p className = 'footer__title'>{footerTitle}</p>
        </footer>
    )
};
Footer.propTypes = {
    footerTitle: PropTypes.string.isRequired
}
export default Footer;