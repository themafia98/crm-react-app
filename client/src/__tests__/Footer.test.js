import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../Components/Footer/Footer';

const FooterTest = renderer.create(
    <Footer footerTitle = 'Test!' />
);

test('Footer test', () => {

    let treeSnapshotFooter = FooterTest.toJSON();
    expect(treeSnapshotFooter).toMatchSnapshot();
});