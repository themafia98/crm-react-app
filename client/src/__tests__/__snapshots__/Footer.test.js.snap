// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer test 1`] = `
<footer>
  <p
    className="footer__title"
  >
    Test!
  </p>
</footer>
`;
